docs-devsite/index.md

```diff
@@ -27,5 +27,5 @@ https://github.com/firebase/firebase-js-sdk
 |  [@firebase/performance](./performance.md#performance_package) | The Firebase Performance Monitoring Web SDK. This SDK does not work in a Node.js environment. |
 |  [@firebase/remote-config](./remote-config.md#remote-config_package) | The Firebase Remote Config Web SDK. This SDK does not work in a Node.js environment. |
 |  [@firebase/storage](./storage.md#storage_package) | Cloud Storage for Firebase |
-|  [@firebase/vertexai-preview](./vertexai-preview.md#vertexai-preview_package) | The Vertex AI For Firebase Web SDK. |
```

Comment:
Can we leave the "Web" bit?

Reply:
Sure, I can revert this

For my own understanding, why do we want to refer to it as the Web SDK when it is also available in non-web environments (Node)?

I guess this isn't a question specific to Vertex AI in Firebase, since I feel like I've seen 'JS SDK' and 'Web SDK' used interchangeably in other places.

cc @hsubox76?